src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -118,6 +118,12 @@ export class ShippingRoute extends Component<ShippingProps> {
                   </Join>
 
                   <Spacer mb={3} />
+
+                  <Link to={`/order2/${order.id}/payment`}>
```

Comment:
I think this needs a check for `!xs`, since the Continue button is displayed on the sidebar (see line 131). I'll add this in #1123.